integration-tests/bats/diff.bats

```diff
@@ -720,6 +720,24 @@ SQL
     [[ "$output" = 'UPDATE `t` SET `val1`=30,`val3`=4 WHERE `pk`=1;' ]] || false
 }
 
+@test "diff: skinny flag only shows row changed" {
```

Comment:
It would be good to add tests for some other types of more advanced changes...

For example, when I tested altering the schema to add a column and then update some of the data in the table, I didn't see the results I expected.  When adding a column with all NULL values, `dolt diff` should show only the schema changes and not report any data changes. This worked with `dolt diff --skinny`, but if there were any real data changes, those were not shown. 

```sql
❯ dolt diff --data
diff --dolt a/t b/t
--- a/t @ nfgsd9f16fb4gt1ismhqjv74it1atri5
+++ b/t @ 8h3s3nvj1kjhe2lofpsq59knldv372m4
+---+----+----+--------+------+
|   | n1 | n2 | s      | s2   |
+---+----+----+--------+------+
| < | 2  | 2  | two    | NULL |
| > | 2  | 2  | two    | wha? |
| < | 3  | 3  | three  | NULL |
| > | 3  | 3  | purple | NULL |
+---+----+----+--------+------+

❯ dolt diff --data --skinny
diff --dolt a/t b/t
--- a/t @ nfgsd9f16fb4gt1ismhqjv74it1atri5
+++ b/t @ 8h3s3nvj1kjhe2lofpsq59knldv372m4
```

In the example above, I would expect the `--skinny` output to include the same output as `dolt diff --data` is showing above. 

I would add some tests for covering adding columns and removing columns, mixed in with updating data in the table to make sure we're testing some more non-basic cases. You might also consider other updates like inserting and deleting rows, just to round the tests out.